src/main/java/randoop/contract/CompareToAntiSym.java

```diff
@@ -0,0 +1,67 @@
+package randoop.contract;
+
+import randoop.Globals;
+
+/**
+ * The contract: Reversing a compareTo on two objects should result in equal values of opposite signs
+ * <code>Math.signum(x0.compareTo(x1)) == -Math.signum(x1.compareTo(x0))</code>.
+ */
+public class CompareToAntiSym implements ObjectContract {
```

Comment:
Okay! I changed it to CompareToAntiSymmetric. I was worried that it might be "too long." Is there a general guideline as to when to use abbreviations then?